src/main/java/frc/robot/scoreStates/ClimbState.java

```diff
@@ -41,7 +45,11 @@ public Command driveToClosestOptimalPoint() {
     public Command score() {
         return Commands.sequence(
                 driveToClosestOptimalPoint(),
-                Commands.runOnce(() -> SwerveDrive.getInstance().lock()),
-                elevator.setHeight(CommandGroupsConstants.END_CLIMB_HEIGHT));
+                Commands.runOnce(() -> SwerveDrive.getInstance().lock()));
+    }
+
+    @Override
+    public Command finalizeScore() {
+        return ScoreState.super.finalizeScore();
```

Comment:
Same here about deleting this implementation.

Reply:
b6110bb35c20abf310b1f31b2097ec84e007a7ad